code_src/staking/dot/bounder/boundModule.py

```diff
@@ -0,0 +1,181 @@
+        {'name': 'value', 'type': 51, 'typeName': 'BalanceOf<T>', 'docs': []},
+        {'name': 'payee', 'type': 152, 'typeName': 'RewardDestination<T::AccountId>', 'docs': []}],
+        'documentation': 'Take the origin account as a stash and lock up `value` of its balance. `controller`will
+        be the account that controls it.
+        `value` must be more than the `minimum_balance` specified by`T::Currency`.
+        The dispatch origin for this call must be _Signed_ by the stash account.
+        Emits `Bonded`.
+        # <weight>
+        - Independent of the arguments. Moderate complexity.
+        - O(1).\n- Three extra DB entries.
+        NOTE:Two of the storage writes (`Self::bonded`, `Self::payee`) are _never_ cleaned
+        unless the `origin` falls below _existential deposit_ and gets removed as dust.
+        # </weight>',
+        'module_prefix': 'Staking','module_name': 'Staking', 'spec_version': 9122}
+        :return:
+        """
+        if not reward_address:
+            reward_address = controller_addr
+        try:
+            call = active_substrate.compose_call(
+                call_module=self.call_module,
+                call_function='bond',
+                call_params={
+                    'controller': controller_addr,
+                    'value': value,
+                    'payee': reward_address
+                }
+            )
+
+            return call
+
+        except NotImplementedError as e:
+            self.logger.error(e)
+            sys.exit(0)
+
+    def unBound(self, active_substrate, value):
+        self.logger.info("start unBound script.")
+        """
+        {'call_name': 'unbond', 'call_args':
+         [{'name': 'value', 'type': 51, 'typeName': 'BalanceOf<T>', 'docs': []}],
+         'documentation': 'Schedule a portion of the stash to be unlocked ready for transfer out after the bond
+         period ends. If this leaves an amount actively bonded less than
+         T::Currency::minimum_balance(), then it is increased to the full amount.
+         The dispatch origin for this call must be _Signed_ by the controller, not the stash.
+         Once the unlock period is done, you can call `withdraw_unbonded` to actually move
+         the funds out of management ready for transfer.
+         No more than a limited number of unlocking chunks (see `MAX_UNLOCKING_CHUNKS`)
+         can co-exists at the same time. In that case, [`Call::withdraw_unbonded`]need
+         to be called first to remove some of the chunks (if possible).
+         If a user encounters the `InsufficientBond` error when calling this extrinsic,
+         they should call `chill` first in order to free up their bonded funds.
+         Emits `Unbonded`.See also [`Call::withdraw_unbonded`].',
+        'module_prefix': 'Staking', 'module_name': 'Staking', 'spec_version': 9122}
+        :return:
+        """
+
+        call = active_substrate.compose_call(
```

Comment:
can we put a try catch here as well

Reply:
try catch yes i will try to read code source of the compose_call function and get all possible exceptions.